CONTRIBUTING.md

```diff
@@ -18,8 +18,8 @@ Before we begin, if you don't already have, download [git](https://git-scm.com/d
  - Open the terminal and navigate to the directory where you wish to clone the project using   
     `cd <Address-to-the-directory>`
  - Clone the project and navigate to working directory by typing the commands    
-`git clone https://github.com/<github_account_name>/wayback-machine-chrome.git`   
- `cd wayback-machine-chrome`
+`git clone https://github.com/<github_account_name>/wayback-machine-webextension.git`   
```

Comment:
Please also change the name in the 17th line. Thank you

Reply:
Ohh yaa..I'll do that.

Reply:
Done!